Explorer/Assets/DCL/Chat/ChatController.cs

```diff
@@ -94,6 +96,7 @@ public class ChatController : ControllerBase<ChatView, ChatControllerShowParams>
         private CancellationTokenSource errorNotificationCts = new();
         private CancellationTokenSource memberListCts = new();
         private CancellationTokenSource isUserAllowedCts;
+        private CancellationTokenSource isUserAllowedCts2; // We can't reuse the same
```

Comment:
As a suggestion: I would rename these token sources as something like `isUserAllowedInInitialization` and `isUserAllowedInBusSubscription`. It would give a bit more of information than simply using the same variable name with the suffix `2`.